src/test/isBooleanConstraint.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai'
+import { IsBooleanTrueConstraint, IsBooleanTrue } from '../main/isBooleanTrue'
+import { validateSync } from 'class-validator'
+
+class BooleanTrueTest {
+  @IsBooleanTrue()
+  boolValue?: boolean
+
+  constructor (boolValue?: boolean) {
+    this.boolValue = boolValue
+  }
+}
+
+describe('IsBooleanConstraint', () => {
```

Comment:
Can we please rename this file `src/test/isBooleanConstraint.ts` to `src/test/isBooleanTrue.spec.ts`? and this describe to `describe('IsBooleanTrue',...`